automatic/bluegriffon/tools/chocolateyInstall.ps1

```diff
@@ -1,16 +1,16 @@
 ﻿$ErrorActionPreference = 'Stop'
 
+$toolsPath = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
```

Comment:
Always wondered why people put this in expression when this is perfectly correct:

```
$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
```

Reply:
Honestly, I don't know.
I got so used to seeing it that way that it became a habit (and also because I have it that way in my migration script).

Doesn't really matter though IMO.

Reply:
It matters due to '2.2 Code style'.

Reply:
There isn't anything that conflicts with [2.2 Code style](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md#22-code-style), while it isn't needed and may not correspond to your own coding style, there is still nothing wrong with having it this way.

And just in case you haven't noticed, that line is also the same as the one you get when you run `choco new`

Reply:
More complicated code is less readable then less complicated code. 
Choco internals has nothing to do with standards on this repository.

But whatever really. :)

Reply:
Something to do with PowerShell v2 IIRC.